data/sql/updates/pending_db_world/rev_1625500923667306000.sql

```diff
@@ -0,0 +1,26 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1625500923667306000');
+
+/* Add new gossip */
+INSERT INTO `npc_text`(`ID`,`text0_0`,`text0_1`,`BroadcastTextID0`,
+`lang0`,`Probability0`)
+VALUES
+(50031,
+"Alas $N, I am much too busy to talk. As I mentioned, I need time to think on the situation at hand.
+
+I wish you luck in your travels. Good day.",
+"Alas $N, I am much too busy to talk. As I mentioned, I need time to think on the situation at hand.
+
+I wish you luck in your travels. Good day.",0,0,0);
```

Comment:
As long as this comes out correct in-game I am fine with it.

Usually I use $B for Linebreak.

Reply:
@Kitzunu Thanks for your review. I rewrote gossip with $B as you recommended